### PR TITLE
[ts-sdk] Minimal docs for release

### PR DESCRIPTION
**Pull request by wkozyra95**

### CODE REVIEW

File: ts/live-compositor/README.md [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
 
-Live Components should not be mixed with other React or React Native components, but you can still use hooks like `useState` from React.
+Live Compositor components should not be mixed with other React or React Native components, but you can still use hooks like `useState`/`useEffect` from React.
+
+## Getting started
+
+To try compositor generate started project by running:
+
+```
+npm create live-compositor
```
Comment (wkozyra95) [inline ts/live-compositor/README.md:12]: not sure if we want to provide one option or all of them

File: ts/create-live-compositor/src/options.ts [hunk 2]
```diff
@@ -75,11 +75,23 @@ export async function resolveBrowserOptions(): Promise<BrowserOptions> {
...
   ] as const);
   return {
     type: 'node',
     templateName,
   };
 }
+
+export async function resolvePackageManager(): Promise<PackageManager> {
+  const nodeUserAgent = process.env.npm_config_user_agent;
+  if (nodeUserAgent?.startsWith('pnpm')) {
```
Comment (wkozyra95) [inline ts/create-live-compositor/src/options.ts:89]: If you run yarn or pnpm create then you won't be prompted about package manager

**Comment by wkozyra95** [comment]
This is not everything needed for release, Let me know what you think is missing

File: ts/@live-compositor/node/README.md [hunk 3]
```diff
@@ -30,19 +30,36 @@ async function run() {
...
+## License
+
+`@live-compositor/node` is MIT licensed, but internally it is downloading and using Live Compositor server that is licensed
+under [Business Source License 1.1](https://github.com/software-mansion/live-compositor/blob/master/LICENSE).
+
+## LiveCompositor is created by Software Mansion
+
+[![swm](https://logo.swmansion.com/logo?color=white&variant=desktop&width=150&tag=live-compositor-github 'Software Mansion')](https://swmansion.com)
+
+Since 2012 [Software Mansion](https://swmansion.com) is a software agency with experience in building web and mobile apps. We are Core React Native Contributors and experts in dealing with all kinds of React Native issues. We can help you build your next dream product – [Hire us](https://swmansion.com/contact/projects?utm_source=live-compositor&utm_medium=readme).
```
Comment (noituri) [inline ts/@live-compositor/node/README.md:65]: Maybe it would be worth mentioning something related to multimedia

File: ts/README.md [hunk 4]
```diff
@@ -5,3 +5,9 @@
 - `live-compositor` - Provide core React components that can be used to define a composition of a video stream.
 - `@live-compositor/node` - Provides API to create and manage LiveCompositor instances for Node.js environment.
 - `@live-compositor/core` - Shared runtime-independent implementation that is used by packages like `@live-compositor/node`.
+- `@live-compositor/browser-render` - Rendering implementation from LiveCompositor compiled to WASM and using WebGL as rendering backend.
```
Comment (noituri) [inline ts/README.md:8]:
```suggestion
- `@live-compositor/browser-render` - Rendering implementation from LiveCompositor compiled to WASM and uses WebGL as rendering backend.
```

File: ts/@live-compositor/node/README.md (hunk 3 above)
Comment (WojciechBarczynski) [reply ts/@live-compositor/node/README.md:65]: +1

File: ts/live-compositor/README.md [hunk 5]
```diff
@@ -20,28 +39,23 @@ function ExampleApp() {
...
+## License
+
+`live-compositor` package is MIT licensed, but it is only useful when used with Live Compositor server that is licensed
+under [Business Source License 1.1](https://github.com/software-mansion/live-compositor/blob/master/LICENSE).
+
+## LiveCompositor is created by Software Mansion
+
+[![swm](https://logo.swmansion.com/logo?color=white&variant=desktop&width=150&tag=live-compositor-github 'Software Mansion')](https://swmansion.com)
+
+Since 2012 [Software Mansion](https://swmansion.com) is a software agency with experience in building web and mobile apps. We are Core React Native Contributors and experts in dealing with all kinds of React Native issues. We can help you build your next dream product – [Hire us](https://swmansion.com/contact/projects?utm_source=live-compositor&utm_medium=readme).
```
Comment (WojciechBarczynski) [inline ts/live-compositor/README.md:61]: Focus on multimedia

**Review by WojciechBarczynski — APPROVED** [review]

**Pull request merged**